--- pcre-7.1/pcreposix.h.orig	2007-04-04 16:04:40.000000000 +0200
+++ pcre-7.1/pcreposix.h	2007-05-01 12:56:05.456674588 +0200
@@ -111,8 +111,13 @@
 
 #ifdef _WIN32
 #  ifndef PCREPOSIX_STATIC
+#   ifdef PCRE_EXP_DECL
+#    define PCREPOSIX_EXP_DECL PCRE_EXP_DECL
+#    define PCREPOSIX_EXP_DEFN PCRE_EXP_DEFN
+#   else
 #    define PCREPOSIX_EXP_DECL extern __declspec(dllimport)
 #    define PCREPOSIX_EXP_DEFN  __declspec(dllimport)
+#   endif
 #  endif
 #endif
 
